### DISCUSSION TITLE
Export settings?

### DISCUSSION THREAD

**Discussion by realm174** [lllyasviel/Fooocus#2125] [Q&A] [answered]
Hello, I know we can modify settings.txt so that Fooocus loads some default settings. I've gone through the tutorial file, but somehow, I always seem to manage to break things when i try to manually edit the file to reflect my current settings. Is there a way to export the current settings to settings.txt once we're satisfied with the changes we've set?  I am aware of the button to export parameters from the log.html file, but that's something i would have to do each time, whereas settings.txt would be automatically loaded if I understand correctly.  Suggestions?

**Comment by mashb1t** [comment]
No there is not. Feel free to post your config file + your terminal output here if you're facing issues.
Please make sure that you need to have no comma `,` at the end o the last line.

**Comment by Irmagon** [comment, marked as answer]
Fooocus alvays starts with filled ALL parameters
Parameters reads from files in this orders:
presets\presets file -> config.txt -> modules\config.py

Right way -  create custom preset with u needed parameters...

**Comment by eddyizm** [comment]
As mentioned above, a preset your best bet.

**Comment by realm174** [comment]
Much appreciated, thank you everyone. I'll go with the preset.
